build/docs-misc.leafdoc

```diff
@@ -54,7 +54,10 @@ for this section of the documentation, please edit docs-misc.leafdoc in the buil
 | drawError | Object | [See code](https://github.com/Leaflet/Leaflet.draw/blob/master/src/draw/handler/Draw.Polyline.js#L10) | Configuration options for the error that displays if an intersection is detected.
 | guidelineDistance | Number | `20` | Distance in pixels between each guide dash.
 | shapeOptions | [Leaflet Polyline options](http://leafletjs.com/reference.html#polyline-options) | [See code](https://github.com/Leaflet/Leaflet.draw/blob/master/src/draw/handler/Draw.Polyline.js#L20) | The options used when drawing the polyline/polygon on the map.
-| metric | Bool | `true` | Determines which measurement system (metric or imperial) is used.
+| showLength | Bool | `true` | Show the length of the drawn line. **The area is only approximate and become less accurate the larger the polygon is.**
```

Comment:
"The area is only approximate and become less accurate the larger the polygon is."

Ik denk niet dat dit functioneel acceptabel is. Er zijn ooit hele discussies geweest over de nauwkeurigheid van Leaflet.measure. Dit is dan een stap terug.